docs/installing.md

```diff
@@ -66,9 +66,31 @@ You can now:
 ansible-playbook -i inventory/hosts setup.yml --tags=ensure-matrix-users-created,start
 ```
 
+## Create your user account
+
+ℹ️ *You can skip this step if you have installed a server and imported old data to it.*
+
+As you have configured your brand new server and the client, you need to **create your user account** on your Matrix server.
+
+After creating the user account, you can log in to it with [Element Web](configuring-playbook-client-element-web.md) that this playbook has installed for you at this URL: `https://element.example.com/`.
+
+To register a user via this Ansible playbook, run the command below on your local computer.
+
+**Notes**:
+- Before running it, make sure to edit `YOUR_USERNAME_HERE` and `YOUR_PASSWORD_HERE`
+- In the command below, `YOUR_USERNAME_HERE` is just a plain username (like `john`), not your full `@user:example.com` identifier
+
+```sh
+ansible-playbook -i inventory/hosts setup.yml --extra-vars='username=YOUR_USERNAME_HERE password=YOUR_PASSWORD_HERE admin=<yes|no>' --tags=register-user
+
+# Example: `ansible-playbook -i inventory/hosts setup.yml --extra-vars='username=john password=secret-password admin=yes' --tags=register-user`
+```
+
+For more information, see the documentation for [registering users](registering-users.md).
+
 ## Finalize the installation
 
-Now that services are running, you need to **finalize the installation process** by [delegating (redirecting) the server](howto-server-delegation.md) to your base domain (`example.com`).
+Now you've configured Matrix services and your user account, you need to **finalize the installation process** by [delegating (redirecting) the server](howto-server-delegation.md) to your base domain (`example.com`).
```

Comment:
```suggestion
Now you've configured Matrix services and your user account, you need to **finalize the installation process** by  [setting up Matrix delegation (redirection)](howto-server-delegation.md), so that your Matrix server (`matrix.example.com`) can present itself as the base domain (`example.com`) in the Matrix network.
```

Your original sentence ("delegating the server to the base domain") is kind of misleading, I think.

I've also tried to avoid using words that may indicate that "the base domain is redirected to the Matrix server", because people may think they need to redirect all of `example.com` to `matrix.example.com` or something, whereas in reality they only need to set up a few file redirects (or a DNS SRV record).

Reply:
The word `delegation` is really a technical term, and I indeed did not quite get it right when I was translating the word. I would say the term deserves an FAQ entry ("People are talking about 'delegation', but what is it?")